### DISCUSSION TITLE
3.0.x.x vs 3.0.3.9

### DISCUSSION THREAD

**Discussion by WebSpectrum** [opencart/opencart#13700] [General]
What exactly is difference between 3.0.x.x and 3.0.3.9 ?

**Comment by mhcwebdesign** [comment]
3.0.x.x is based on 3.0.3.9, but with some further bugfixes.

**Comment by felipoantonoff** [comment]
@mhcwebdesign Do you know if Opencart 3 will continue to be maintained alongside Opencart 4 or will it continue until Opencart 4 becomes more mature and more used?

**Reply by mhcwebdesign** [reply]
Yes, OpenCart 3 will be maintained for quite a while. In fact, the upcoming version 3.0.4.0 is about ready to be tagged and released.

**Reply by WebSpectrum** [reply]
Hi - I have installed 3.0.x.x with some extensions, is it straightforward to upgrade to the 3.0.4.0 release?

**Reply by mhcwebdesign** [reply]
@WebSpectrum : yes, it should be easy enough to go to 3.0.4.0 from 3.0.x.x.

**Reply by felipoantonoff** [reply]
> Yes, OpenCart 3 will be maintained for quite a while. In fact, the upcoming version 3.0.4.0 is about ready to be tagged and released.

Great, good to know, thanks for the information.
